alpha/apps/kaltura/lib/storage/kStorageExporter.php

```diff
@@ -39,7 +40,9 @@ public function objectChanged(BaseObject $object, array $modifiedColumns)
 			foreach($externalStorages as $externalStorage)
 			{
 				if($externalStorage->getTrigger() == StorageProfile::STORAGE_TEMP_TRIGGER_MODERATION_APPROVED)
-					self::exportEntry($object, $externalStorage);
+				{
+					self::tryExportEntry($object, $externalStorage);
```

Comment:
No need to wrap with try and catch as you already know that these profiles are automatic.